protocol.go

```diff
@@ -154,6 +154,11 @@ func (p *Conn) RemoteAddr() net.Addr {
 	return p.header.SourceAddr
 }
 
+// Conn returns is the underlying connection
+func (p *Conn) Conn() net.Conn {
```

Comment:
`p.Conn.Conn()` is stuttering. IIRC I had suggested in some other past PR to have this named `Raw()`. I don't remember any disagreement. Would you be ok w/ change it?

Also, can you please document further (as function comment) what this function can be used for and any concerns about it, eg buffered data?

Reply:
Done.